packages/ui/src/components/vuestic-components/va-hover/VaHover.vue

```diff
@@ -1,35 +1,42 @@
-<script>
-export default {
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+
+@Component({
   name: 'VaHover',
+})
+export default class VaHover extends Vue {
+  @Prop({
+    type: Boolean,
+    default: false,
+  })
+  readonly disabled!: boolean
+
+  @Prop({
+    type: Boolean,
+  })
+  readonly value!: boolean
+
   data () {
     return {
       active: false,
```

Comment:
Data shouldn't look like that.
Please check in other places, I might have skipped.